lib/server.js

```diff
@@ -41,27 +41,39 @@ var Server = function(server, path, services, wsdl, options) {
   if (path[path.length - 1] !== '/')
     path += '/';
   wsdl.onReady(function(err) {
-    var listeners = server.listeners('request').slice();
-
-    server.removeAllListeners('request');
-    server.addListener('request', function(req, res) {
-      if (typeof self.authorizeConnection === 'function') {
-        if (!self.authorizeConnection(req)) {
-          res.end();
-          return;
+    if(typeof server.route === 'function' && typeof server.use === 'function'){
```

Comment:
is there any other possibility to check if the passed in server instance is an `express` app or is this check reliable from at least 4.x upwards?